src/derivative_operators/multi_dim_bc_operators.jl

```diff
@@ -51,7 +51,7 @@ struct MultiDimDirectionalBC{T<:Number, B<:AtomicBC{T}, D, N, M} <: MultiDimensi
 end
 
 struct ComposedMultiDimBC{T, B<:AtomicBC{T}, N,M} <: MultiDimensionalBC{T, N}
-    BCs::Vector{Array{B, M}}
+    BCs::Vector{Array{B, M}} # Why isn't this a vector of MultiDimBCs?
```

Comment:
Looks like it's a vector of `AtomicBC`. I'll keep reading to see if that makes sense.

Reply:
Yeah seems like that works.